src/components/Proposal/Create/CreateProposalPage.tsx

```diff
@@ -14,30 +14,32 @@ import { BreadcrumbsItem } from "react-breadcrumbs-dynamic";
 import { connect } from "react-redux";
 import { IRootState } from "reducers";
 import { RouteComponentProps } from "react-router-dom";
-import { CrxRewarderComponentType, getCrxRewarderComponent, rewarderContractName } from "components/Scheme/ContributionRewardExtRewarders/rewardersProps";
-import CreateContributionRewardProposal from "components/Proposal/Create/SchemeForms/CreateContributionRewardProposal";
-import { schemeName } from "lib/schemeUtils";
+import { CrxRewarderComponentType, getCrxRewarderComponent, rewarderContractName } from "components/Plugin/ContributionRewardExtRewarders/rewardersProps";
+import CreateContributionRewardProposal from "components/Proposal/Create/PluginForms/CreateContributionRewardProposal";
+import { pluginName } from "lib/pluginUtils";
 import * as css from "./CreateProposal.scss";
+import CreatePluginManagerProposal from "./PluginForms/CreatePluginManagerProposal";
+import { of } from "rxjs";
 
 type IExternalProps = RouteComponentProps<any>;
 
 interface IExternalStateProps {
   daoAvatarAddress: string;
   history: History;
-  schemeId: string;
+  pluginId: string;
 }
 
 interface IStateProps {
   createCrxProposalComponent: any;
 }
 
-type IProps = IExternalProps & IExternalStateProps & ISubscriptionProps<ISchemeState>;
+type IProps = IExternalProps & IExternalStateProps & ISubscriptionProps<AnyPlugin>;
```

Comment:
We have a lint rule that all interfaces must start with 'I'.  `AnyPlugin` looks like an interface here.

Reply:
It's an abstract base class: https://github.com/daostack/arc.js/blob/ad9d4c49dd4d25170a6df7e37d5c084f81cb57aa/src/plugins/utils.ts#L105